### ISSUE TITLE
Add support for ORDER BY ${field} NULLS FIRST|LAST

### ISSUE THREAD

**Issue by boh1996** [knorm/knorm#84]
For ordering PostgresSQL supports ORDER BY exp NULLS FIRST etc, see: https://www.postgresql.org/docs/8.3/static/queries-order.html . 
But making a query like that in knorm is not possible, because Knorm validates the direction strict.

**Comment by joelmukuthu** [comment]
Makes sense. But as far as I can I see, this is implicitly supported since knorm also allows arbitrary sort strings. Try `orderBy('column_name asc nulls first')`. Note that, because that’s raw sql, you’ll have to use column-names and not field-names. LMK :)

**Comment by joelmukuthu** [comment]
Cool. Let’s keep this open though because it’d still be nice to support `orderBy({ field: 'asc nulls first' })` and `orderBy(sql('column asc nulls first'))` when passing raw sql, which I believe would fail atm.
